x-pack/plugins/triggers_actions_ui/public/application/hooks/use_load_rule_alerts_aggregations.ts

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
I will prefer to see a folder with multiple files (`types.ts`, `helpers,ts` ...) It will help the unit test and the readability

Reply:
@XavierM, Okay. But it's not the followed practice in the `hooks` folder. i.e., each hook is a single file, and inside it has all related types and helpers. For testing, this related stuff (types, helpers, etc..) are exported.

What do you think?